spec/datadog/tracing/distributed/helpers_spec.rb

```diff
@@ -73,12 +73,9 @@
     context 'when value is' do
       [
         [nil, nil],
+        ['not a number', nil],
         ['0', nil],
-        ['value', nil],
-        ['867-5309', nil],
```

Comment:
This test in particular I think was useful because:

```
[4] pry(main)> '867-5309'.to_i
=> 867
```

and actually we have extra code to make it `nil`

(Same note applies below)

Reply:
Restored as `'1.to_i returns 1'`, thank you!